genkit-tools/cli/config/firebase.index.ts.template

```diff
@@ -26,7 +26,8 @@ export const menuSuggestionFlow = onFlow(
     }),
   },
   async (subject) => {
-    const prompt = `Suggest an item for the menu of a ${subject} themed restaurant`;
+    const prompt = `Suggest an item for the menu of a
+     ${subject} themed restaurant`;
```

Comment:
is this intentional?

Reply:
Yes, but open to a different way of solving the max-len of 80 lint error for Firebase functions

Reply:
I'd recommend this instead:
```ts
    const prompt =
        `Suggest an item for the menu of a ${subject} themed restaurant`;
```